### PR TITLE
binary translations page not display zulu language 

### PR DESCRIPTION
**Pull request by rra0137**
Would you please check this screen shot https://prnt.sc/mbdcf6, it's russian language

### CODE REVIEW

**Comment by rra0137** [comment]
@chcg , @donho would you please answer my query why the Zulu language does not display on binary translation page its https://prnt.sc/mbdcf6, it's the Russian language

and Zulu language is accepted by notepad plus

File: PowerEditor/src/localizationString.h [hunk 1]
```diff
@@ -115,5 +115,6 @@ LocalizationSwitcher::LocalizationDefinition localizationDefs[] =
 	{TEXT("Corsu"), TEXT("corsican.xml")},
 	{TEXT("Brezhoneg"), TEXT("breton.xml")},
 	{TEXT("کوردی‬"), TEXT("kurdish.xml")},
-	{TEXT("Pig latin"), TEXT("piglatin.xml")}
-};
\ No newline at end of file
+	{TEXT("Pig latin"), TEXT("piglatin.xml")},
+        {TEXT("Zulu"), TEXT("zulu.xml")}
```
Comment (chcg) [inline PowerEditor/src/localizationString.h]: Spaces instead of Tab

File: PowerEditor/src/Parameters.cpp [hunk 2]
```diff
@@ -3827,7 +3827,9 @@ generic_string NppParameters::getLocPathFromStr(const generic_string & localizat
 		return TEXT("vietnamese.xml");
 	if (localizationCode == TEXT("cy-gb"))
 		return TEXT("welsh.xml");
-
+        if (localizationCode == TEXT("zu") || localizationCode == TEXT("zu-za"))
```
Comment (chcg) [inline PowerEditor/src/Parameters.cpp]: @donho This change is not merged yet.

**Comment by chcg** [comment]
@rra0137 You are working directly on the master branch of your fork from N++ git repository. It would be better to use branches for your work and leave the master unchanged.

See e.g. https://gist.github.com/vlandham/3b2b79c40bc7353ae95a

File: PowerEditor/src/Parameters.cpp (hunk 2 above)
Comment (rra0137) [reply PowerEditor/src/Parameters.cpp]: @donho yes not merged yet

File: PowerEditor/src/localizationString.h (hunk 1 above)
Comment (rra0137) [reply PowerEditor/src/localizationString.h]: Check this already done https://prnt.sc/mbv07w

Comment (rra0137) [reply PowerEditor/src/localizationString.h]: @chcg this is the link https://notepad-plus-plus.org/contribute/binary-translations.html where not display Zulu language yet it's the Russian language

**Comment by chcg** [comment]
@donho Zulu is missing here https://notepad-plus-plus.org/contribute/binary-translations.html, so probably the flag of south africa, see https://en.wikipedia.org/wiki/Zulu_people.

**Comment by rra0137** [comment]
> 
> 
> @donho Zulu is missing here https://notepad-plus-plus.org/contribute/binary-translations.html, so probably the flag of south africa, see https://en.wikipedia.org/wiki/Zulu_people.

@chcg ,@donho Would you please check this and please check above comment from @chcg ?

**Pull request closed without merging**